drivers/serial/uart_nrfx_uart.c

```diff
@@ -191,8 +191,6 @@ static void event_txdrdy_clear(void)
  *
  * @param dev UART device struct
  * @param baudrate Baud rate
- *
- * @return N/A
```

Comment:
you might want to note that this function returns -EINVAL for a bad baudrate, 0 on success.

Reply:
I got a request for a re-review, but I don't see where my comments were addressed for this or intc_plic.c

Reply:
Sorry. Missed this one. Fixed now.